src/timeMachine/actions/queries.ts

```diff
@@ -15,22 +15,16 @@ import {notify} from 'src/shared/actions/notifications'
 import {hydrateVariables} from 'src/variables/actions/thunks'
 
 // Constants
-import {
-  rateLimitReached,
-  resultTooLarge,
-  updateAggregateType,
-} from 'src/shared/copy/notifications'
+import {rateLimitReached, resultTooLarge} from 'src/shared/copy/notifications'
 
 // Utils
 import fromFlux from 'src/shared/utils/fromFlux'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 import {buildUsedVarsOption} from 'src/variables/utils/buildVarsOption'
 import {findNodes} from 'src/shared/utils/ast'
-import {isAggregateTypeError} from 'src/utils/aggregateTypeErrors'
 import {event} from 'src/cloud/utils/reporting'
 import {asSimplyKeyValueVariables, hashCode} from 'src/shared/apis/queryCache'
 import {filterUnusedVarsBasedOnQuery} from 'src/shared/utils/filterUnusedVars'
-import {getAggregateTypeErrorButton} from 'src/shared/components/notifications/NotificationButtons'
```

Comment:
`getAggregateTypeErrorButton` is only being used here, so you can delete that method from `src/shared/components/notifications/NotificationButtons`.